### ISSUE TITLE
a "bug"? in the cpo method

### ISSUE THREAD

**Issue by xzhang2523** [ajlangley/cpo-pytorch#8]
reward_advs -= reward_advs.mean()
            reward_advs /= reward_advs.std()
            cost_advs -= \textbf{reward_advs}.mean()
            cost_advs /= cost_advs.std()

I guess on line 3, it should be mean of the cost?

**Comment by ajlangley** [comment]
You are right. If you want to make any changes to the repo and make a PR, I'm happy to merge it in.
